react/ButtonGroup/ButtonGroup.js

```diff
@@ -0,0 +1,17 @@
+import styles from './ButtonGroup.less';
+import React, { PropTypes } from 'react';
+
+export default function ButtonGroup({ children }) {
+  return (
+    <div>{children.map(child => React.cloneElement(child, { className: styles.button }))}</div>
```

Comment:
The issue with this implementation is that you might potentially clash with margins on child elements, whereas container elements don't have to worry about this.